docs/setup-instructions.md

```diff
@@ -0,0 +1,51 @@
+## Setup instructions
+
+#### Prerequisites
+ - Android Studio or any IDE to open Flutter project
+ - JDK
+
+#### Installations
+1. Install Flutter by following instructions from [flutter.dev](https://flutter.dev).To summarise:
+  - Select the appropriate operating system
+  - Download the flutter sdk to a preferred location on your local system.
+2.Fork and clone the [mentorship-flutter](https://github.com/anitab-org/mentorship-flutter) repository to your local machine.
+3.Make sure to install the **Flutter** and **Dart** plugins.
+  - If mentorship Flutter is the first project that you will be viewing in Android Studio then:
+      - Start Android Studio
+      - Open Plugin Preferences
+      - Browse repositories and search for flutter
+      - Install and click yes to install Dart as well if prompted.
+  - Flutter and dart can also be installed after opening a project.
+      - Go to File menu -> Settings -> plugins
+      - Search for the plugin. In this case it would be Flutter and Dart. Install them if not done yet and click Apply.
+
+      ![FlutterDartPlugin](images/flutter_dart_plugin.gif)
+
+#### Developer's locally
```

Comment:
```suggestion
## Local Development Setup
```
I didn't quite understood what "Developer's locally" meant, what do you think of my suggestion?